lib/addons/p5.dom.js

```diff
@@ -58,12 +58,23 @@
    * </code></div>
    *
    */
-  p5.prototype.select = function (e) {
+  p5.prototype.select = function (e, p) {
     var res;
     var str;
+    var container = document;
+    if (typeof p === 'string'){
+      if(p[0] !== '#'){
+        container = document.getElementById(p);
+      }else {
+        p = p.slice(1);
+        container = document.getElementById(p);
+      }
+    }else if (typeof p === 'object'){
+      container = document.getElementById(p.elt.id);
+    }
```

Comment:
I think we should support one more case, which is passing in an html dom element direct from the page. that is, something obtained manually via getElementById:
```javascript
var elt = document.getElementById('container');
select("#blah", elt);
```

the check for this case would be:
```javascript
} else if (p instanceof HTMLElement) {
  container = p;
}